### ISSUE TITLE
Make segment images size fill the containing canvas

### ISSUE THREAD

**Issue by mlacif1** [zarocknz/javascript-winwheel#26]
Hello.
Would it be possible to make the segment images full size even if the passed in image is too small?
'segments'          :                    // Define segments including image and text.
                [
                   {'image' : 'images/img.png',  'text' : ''},
                   {'image' : 'images/img.png',   'text' : ''},
                   {'image' : 'images/img.png',  'text' : ''},
                   {'image' : 'images/img.png',  'text' : ''},
                   {'image' : 'images/img.png', 'text' : ''},
                   {'image' : 'images/img.png', 'text' : ''},
                   {'image' : 'images/img.png',  'text' : ''},
                   {'image' : 'images/img.png', 'text' : ''}
                ],

Maybe a flag would be nice, like: scaleSegmentImages ? 

Basically make the segment images fill the containing canvas size.
Thank you :)

**Comment by zarocknz** [comment]
Hi. I suppose so. I'll try and find time this weekend to have an experiment. Cheers.

**Comment by AlexanderYW** [comment]
Hi @zarocknz, I know this is an old issue, but I also would like to see this option, are you still going to implement it? :)
